src/tests/helpers/utm-params.test.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, expect, test } from "vitest";
+import { spyOn } from "@vitest/spy";
+import { autoParseUTMParams } from "../../helpers/utm-params";
+
+describe("autoParseUTMParams", () => {
+  test("should return an empty object if no utm param is set", () => {
```

Comment:
Most other tests in the file so not start with "should".

Reply:
Removed the "should", thanks!